src/views/Organizations/OrganizationById.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-container class="my-12" fluid>
+    <base-card>
+      <resource-availability-table
+        :tableColumns="tableColumnsResources"
+        :resourceItem="organizationDetailsItems"
+        :org="orgName.name"
+      ></resource-availability-table>
+    </base-card>
+  </v-container>
+</template>
+
+<script setup>
+import ResourceAvailabilityTable from "@/components/Table/ResourceAvailabilityTable.vue";
+import { ref, computed, inject, onMounted } from "vue";
+import { useRoute } from "vue-router";
+import { useStore } from "vuex";
+
+const store = useStore();
+const route = useRoute();
+const snackbarProvider = inject("snackbarProvider");
+const organizationDetailsItems = ref([]);
+const tableColumnsResources = computed(() => store.getters["users/getColumns"]);
+const orgName = ref({});
+
+onMounted(async () => {
+  fetchUserOrgs();
```

Comment:
Do we need to do this fetch?

Reply:
yes, so we can have the organizations for the user in the select input

Reply:
We don't have a select input anywhere in this view as far as I can see. We should fetch them in the OrganizationResourceAdd/Transfer/Remove view I guess?